index.html

```diff
@@ -8,17 +8,22 @@
       // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
       var respecConfig = {
         specStatus: "CG-DRAFT",
-        shortName: "beforeinstallprompt",
         editors: [{
           name: "Marcos Cáceres"
-        }]
+        }],
+        github: "WICG/manifest-incubations",
+        shortName: "manifest-incubations",
+        xref: {
+          specs: ["appmanifest"],
+          profile: "web-platform",
+        },
       };
     </script>
   </head>
   <body>
     <section id="abstract">
       <p>
-        This specification does neat stuff.
+        Feature specifications for manifest extensions & incubations.
```

Comment:
Perhaps a little more context here to explain why things might go in here and not the main spec.